projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -130,6 +130,16 @@ public function akismet_logo_replacement_styles() {
 	 * @since 11.0 . Prior to that, this function was located in custom-css-4.7.php (now custom-css.php).
 	 */
 	public static function additional_css_menu() {
+		/*
+		 * Custom CSS for the Customizer is deprecated for block themes as of WP 6.1, so we only expose it with a menu
+		 * if the site already has existing CSS code.
+		 */
+		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
+			$styles = wp_get_custom_css();
```

Comment:
This is just a question, why are we checking `wp_get_custom_css()` instead of `Jetpack_Custom_CSS::get_css()` here? Shouldn't we only be checking the Custom CSS registered with Jetpack?